build.gradle

```diff
@@ -12,10 +12,22 @@ repositories {
 
 dependencies {
     compile group: 'net.java.dev.jna', name: 'jna', version: '4.2.0'
```

Comment:
This should probably be 

    compile 'net.java.dev.jna:jna:4.2.0'

For ease-of-reading and consistency.